src/collections/_documentation/performance-monitoring/configuration/node.md

```diff
@@ -37,9 +37,14 @@ const transaction = Sentry.startTransaction({
 });
 
 setTimeout(() => {
-  transaction.finish();
-  foo();
-}, 200);
+  try {
+    foo();
+  } catch (e) {
+    Sentry.captureException(e);
+  } finally {
+    transaction.finish();    
```

Comment:
I was thinking about doing it like that, but IE has no `finally` afair :/

Reply:
I'd leave it like that thought. Cmon.

Reply:
This is for node

Reply:
Problem solved